pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -552,7 +544,16 @@ func (gce *GCECloud) createTargetPool(svc *v1.Service, name, serviceName, ipAddr
 	return nil
 }
 
-func (gce *GCECloud) updateTargetPool(loadBalancerName string, existing sets.String, hosts []*gceInstance) error {
+func (gce *GCECloud) updateTargetPool(loadBalancerName string, hosts []*gceInstance) error {
+	pool, err := gce.service.TargetPools.Get(gce.projectID, gce.region, loadBalancerName).Do()
```

Comment:
While we're here, should we update this call to the wrapper which will soon be changed to use the new cloud package? https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce_targetpool.go#L26

Reply:
Looks like there are a few other places in `_external` that directly call compute api. We can change them all in a later commit.

Reply:
Good call, created https://github.com/kubernetes/kubernetes/pull/58422 separately to keep this PR clean.